src/devices/Mpu9250/Register.cs

```diff
@@ -0,0 +1,413 @@
+        /// </summary>
+        EXT_SENS_DATA_19 = 0x5C,
+        /// <summary>
+        /// External Sensor Data byte 20
+        /// </summary>
+        EXT_SENS_DATA_20 = 0x5D,
+        /// <summary>
+        /// External Sensor Data byte 21
+        /// </summary>
+        EXT_SENS_DATA_21 = 0x5E,
+        /// <summary>
+        /// External Sensor Data byte 22
+        /// </summary>
+        EXT_SENS_DATA_22 = 0x5F,
+        /// <summary>
+        /// External Sensor Data byte 23
+        /// </summary>
+        EXT_SENS_DATA_23 = 0x60,
+        /// <summary>
+        /// I2C Slave 0 Control Data to Write
+        /// </summary>
+        I2C_SLV0_DO = 0x63,
+        /// <summary>
+        /// I2C Slave 1 Control Data to Write
+        /// </summary>
+        I2C_SLV1_DO = 0x64,
+        /// <summary>
+        /// I2C Slave 2 Control Data to Write
+        /// </summary>
+        I2C_SLV2_DO = 0x65,
+        /// <summary>
+        /// I2C Slave 3 Control Data to Write
+        /// </summary>
+        I2C_SLV3_DO = 0x66,
+        /// <summary>
+        /// I2C Master Delay Control
+        /// </summary>
+        I2C_MST_DELAY_CTRL = 0x67,
+        /// <summary>
+        /// Signal Path Reset
+        /// </summary>
+        SIGNAL_PATH_RESET = 0x68,
+        /// <summary>
+        /// Accelerometer Interrupt Control
+        /// </summary>
+        MOT_DETECT_CTRL = 0x69,
+        /// <summary>
+        /// User Control
+        /// </summary>
+        USER_CTRL = 0x6A,
+        /// <summary>
+        /// Power Management 1
+        /// </summary>
+        PWR_MGMT_1 = 0x6B,
+        /// <summary>
+        /// Power Management 2
+        /// </summary>
+        PWR_MGMT_2 = 0x6C,
+        /// <summary>
+        /// FIFO Count Registers High byte
+        /// </summary>
+        FIFO_COUNTH = 0x72,
+        /// <summary>
+        /// FIFO Count Registers Low byte
+        /// </summary>
+        FIFO_COUNTL = 0x73,
+        /// <summary>
+        /// FIFO Read Write
+        /// </summary>
+        FIFO_R_W = 0x74,
+        /// <summary>
+        /// Who Am I
+        /// </summary>
+        WHO_AM_I = 0x75,
+        /// <summary>
+        /// X-axis Accelerometer Offset Register High byte
+        /// </summary>
+        XA_OFFSET_H = 0x77,
+        /// <summary>
+        /// X-axis Accelerometer Offset Register Low byte
+        /// </summary>
+        XA_OFFSET_L = 0x78,
+        /// <summary>
+        /// Y-axis Accelerometer Offset Register High byte
+        /// </summary>
+        YA_OFFSET_H = 0x7A,
+        /// <summary>
+        /// Y-axis Accelerometer Offset Register Low byte
+        /// </summary>
+        YA_OFFSET_L = 0x7B,
+        /// <summary>
+        /// Z-axis Accelerometer Offset Register High byte
+        /// </summary>
+        ZA_OFFSET_H = 0x7D,
+        /// <summary>
+        /// Z-axis Accelerometer Offset Register Low byte
+        /// </summary>
+        ZA_OFFSET_L = 0x7E
```

Comment:
that's a lot of registers :smile:

Reply:
Yes and even commented 😁 But I have to say it was a good exercise. I may comment as well further even those internal registers. It makes it easier later to check that the description and usage match when reading the code. As sometimes the register name is not the best in the documentation. And as we keep the registers as close as possible to the documentation, it actually not a bad idea to comment all of them.